src/panel/category/CategoryPanel.jsx

```diff
@@ -64,6 +65,8 @@ const CategoryPanel = ({ poiFilters = {}, bbox }) => {
   const { isMobile } = useDevice();
   const { maxPlaces } = useConfig('category');
 
+  usePageTitle(capitalizeFirst(poiFilters.category || poiFilters.query));
```

Comment:
`poiFilters.category` is the value defined by the API and is not translated. Also should `poiFilters.query` really be capitalized ?

Perhaps this logic could be shared with the existing [`getTopBarAppValue()`](https://github.com/Qwant/erdapfel/blob/1d67287fb4d07a9803e715fb794f6dd842369224/src/panel/PanelManager.jsx#L19-L30).

Reply:
I first went into refactoring once again parts of upper app state, but finally I just defined a new function to share this https://github.com/Qwant/erdapfel/pull/1160/commits/925c432320505892c763fcd74f9ab097c828a5eb